packages/meteor-gazelle-wiki/lib/methods.js

```diff
@@ -0,0 +1,18 @@
+Wiki = {};
+
+Meteor.methods({
+  'wiki/createArticle': function (doc) {
+    var article = new Article();
+    article.set('title', doc.title);
+    article.set('body', doc.content);
+    article.set('createdBy', this.userId);
+    article.set('locked', false);
+    article.save();
+  },
+  'wiki/lockArticle': function (article) {
```

Comment:
@ajax-wcd why actually should it be a thing? If an article should be locked its editing permissions should just be put higher.